packages/ckeditor5-dev-tests/tests/bin/create-mgit.json.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { expect } = require( 'chai' );
+
+describe( 'dev-tests/bin/create-mgit-json', () => {
```

Comment:
Would be nice to have a test which checks what happens if passed packages are invalid (do not start with `@ckeditor/ckeditor5` or package is a dev package).